src/main/java/ddeliopoulos/github/gardenjournal/journalentry/JournalService.java

```diff
@@ -0,0 +1,73 @@
+package ddeliopoulos.github.gardenjournal.journalentry;
+
+import ddeliopoulos.github.gardenjournal.journalentry.api.CreateJournalRequest;
+import ddeliopoulos.github.gardenjournal.journalentry.api.GetJournalResponse;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.stereotype.Service;
+import org.springframework.web.server.ResponseStatusException;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@Service
+public class JournalService {
+
+    private final JournalRepository journalRepository;
+
+    @Autowired
+    JournalService(final JournalRepository journalRepository) {
+        this.journalRepository = journalRepository;
+    }
+
+    List<GetJournalResponse> getJournalEntries() {
+        final List<JournalEntry> allJournalEntries = journalRepository.findAll();
+
+        final List<GetJournalResponse> allJournalEntriesMapped = new ArrayList<>(allJournalEntries.size());
+        for (JournalEntry journalEntry : allJournalEntries) {
+            final GetJournalResponse mappedJournalEntry = mapEntityToGetResponse(journalEntry);
+            allJournalEntriesMapped.add(mappedJournalEntry);
+        }
+        return allJournalEntriesMapped;
```

Comment:
Could be more functional:

```suggestion
        return journalRepository.findAll()
                                .stream()
                                .map(this::mapEntityToGetResponse)
                                .collect(Collectors.toList());
```